### DISCUSSION TITLE
Feature Request | text/html file attachments

### DISCUSSION THREAD

**Discussion by robi4513** [Shelex/cypress-allure-plugin#136] [Q&A] [answered]
I found this implementation for an axe pytest integration that uses content type of HTML to render the page [(Allure Framework docs)](https://docs.qameta.io/allure/#:~:text=to.%20Attachments%20of-,HTML,-type%20are%20rendered) within the report. Of the content types offered for `cypress-allure-plugin`, text/css is compatible with .html but will only allow for a html file to be downloaded, not rendered.
Is there a way to add `text/html` as a ContentType?
Or is there already a solution to this that I am overlooking?

**Comment by Shelex** [comment, marked as answer]
Released in v2.32.1
